analyzers/its/expected/Ember-MM/generic.EmberCore.NMT-{84B2143A-D04F-4262-923D-21AEDF86E2B7}-S2259.json

```diff
@@ -2,6 +2,149 @@
 "issues":  [
 {
 "id":  "S2259",
+"message":  "'r.values.FirstOrDefault(Function(y) y.value = v)' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  166,
+"startColumn":  97,
+"endLine":  166,
+"endColumn":  145
+}
+}
+},
+{
+"id":  "S2259",
+"message":  "'conf.Files.FirstOrDefault(Function(y) y.Process = True AndAlso y.Type = "movieindex")' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  298,
+"startColumn":  61,
+"endLine":  298,
+"endColumn":  146
+}
+}
+},
```

Comment:
TP `Name` is a field.
https://github.com/SonarSource/sonar-dotnet/blob/145047ee1ed01316ef730427c7af96f0602ee2d8/analyzers/its/sources/Ember-MM/Addons/generic.EmberCore.NMT/dlgNMTMovies.vb#L298
https://github.com/SonarSource/sonar-dotnet/blob/1711a5f773013925e079d11e32bf0a3f592a6706/analyzers/its/sources/Ember-MM/Addons/generic.EmberCore.NMT/Module.NMT.vb#L207-L208